globus_cli/version.py

```diff
@@ -22,3 +23,22 @@
 
 # app name to send as part of SDK requests
 app_name = 'Globus CLI v{} - Alpha'.format(__version__)
+
+
+# pull down version data from PyPi
+def get_versions():
+    """
+    Wrap in a function to ensure that we don't run this every time a CLI
+    command runs (yuck!)
+
+    Also protects import of `requests` from issues when grabbed by setuptools.
+    More on that inline
+    """
+    # import in the func (rather than top-level scope) so that at setup time,
+    # `requests` isn't required -- otherwise, setuptools will fail to run
+    # because requests isn't installed yet.
+    import requests
+    _version_data = requests.get(
```

Comment:
I'm assuming that you have this prefixed with an underscore because originally it was module-level (i.e. it didn't used to be wrapped in a function). You could get rid of that now if you wanted.